qa/rpc-tests/test_framework/mininode.py

```diff
@@ -558,38 +624,6 @@ def serialize_with_witness(self):
         r += struct.pack("<I", self.nLockTime)
         return r
 
-    # Only serialize with witness when explicitly called for
-    def serialize_with_witness(self):
```

Comment:
the ``serialize_with_witness`` function was duplicated. This removes the second copy as the first one matched the C++ code better